### DISCUSSION TITLE
Game controller forwarding

### DISCUSSION THREAD

**Discussion by divinity76** [rustdesk/rustdesk#6631] [Feature Request]
Would be awesome if we could get game controllers forwarded, for example a Xbox 360 controller.

I don't think this will be an easy thing to implement, I know Google Stadia, Nvidia GeForce Now, and Amazon Luna, cloud gaming services, managed to do it, but they literally had millions for R&D.

Quoting a Google Stadia developer on the subject,
>we support current USB controllers that support the HID standard so any USB controller that you already have will generally work 

(ref https://www.eurogamer.net/digitalfoundry-2019-google-stadia-phil-harrison-majd-bakar-interview )

**Comment by RomainFallet** [comment]
Yes, Rustdesk is so performant that I can play some games that run on my PC from by laptop through Rustdesk. Would love to see this feature too!

**Comment by SevDaisy** [comment]
I cannot AGREE you any more. Before that come to truth, I'm taking `parsec` as replacement. Even that, I'm keeping desire with this feature in RustDesk day and night.

**Comment by IndyHendrickx** [comment]
This would be so cool! Would love to see this feature.
